src/opm/output/eclipse/Summary.cpp

```diff
@@ -218,7 +218,7 @@ inline quantity rate( const fn_args& args ) {
 template< bool injection >
 inline quantity flowing( const fn_args& args ) {
     const auto& wells = args.wells;
-    const auto ts = args.timestep;
+    const auto ts = args.timestep + 1;
```

Comment:
Are you certain this should be `+1` - I would actually think that we want here is the raw `timestep` value; although I agree that will be a change.